lib/datasets/Flights.py

```diff
@@ -1,2 +1,58 @@
-def dummy_dataset():
-    return [1, 2, 3, 4]
+import pandas as pd
```

Comment:
The usual way for multiple people to work together in the same branch would be via PRs. For example, if @majdbar321 started the front end, now instead of the rest committing to the same branch, they would do a PR to Madj's branch. If Madj is ok with the code, he would merge the code into his branch, and will ask the rest of the team to review before emerging to main.

Makes sense?